fuel-merkle/src/sparse/merkle_tree.rs

```diff
@@ -1344,13 +1344,15 @@ mod test {
     #[test]
     fn test_from_set_yields_expected_root() {
         let rng = &mut rand::thread_rng();
-        let gen = || {
+        let generator = || {
```

Comment:
Was this done automatically or did you manually change the name? (just curious)

Reply:
Manually :)